Task 1/matrix search/question

```diff
@@ -0,0 +1,6 @@
+Write an algorithm that searches for a value in an m x n matrix.
```

Comment:
Rename file as question.md to allow markdown syntax

Reply:
Ok